iis-core/src/test/java/eu/dnetlib/iis/core/IntegrationTestPropertiesReader.java

```diff
@@ -38,6 +50,8 @@ public IntegrationTestPropertiesReader() {
 	    
 	    integrationTestProperties.putAll(defaultProperties);
 	    integrationTestProperties.putAll(userProperties);
+	    
+	    createTempPropertiesFile();
```

Comment:
This constructor doesn't look elegant. It implicitly reads properties, creates files. Apart from that - the class is not focused on one thing. Let's talk about it at our meeting.